CHANGELOG.md

```diff
@@ -845,35 +960,34 @@ About
 
 Ginkgo 1.0.0 is brought to you by:
 
-**Karlsruhe Institute of Technology**, Germany  
```

Comment:
the following author list should contain the two spaces in the end for changing line